admin/proc-rhdh-deploy-eks-using-helm.adoc

```diff
@@ -0,0 +1,177 @@
+    annotations:
+      # alb because we are using ALB.
+      # But adjust if using a different Ingress Controller
+      # and remove the 'alb.*' annotations accordingly.
+      kubernetes.io/ingress.class: alb
+
+
+      # Below annotation is to specify if the loadbalancer is "internal" or "internet-facing"	   
+      alb.ingress.kubernetes.io/scheme: internet-facing
+
+
+      # TODO: Using an ALB HTTPS Listener requires a certificate for your own domain. Fill in the ARN of your certificate, e.g.:
+      # alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-west-2:xxxx:certificate/xxxxxx
+
+
+	# TODO: The HTTPS listener below requires setting the certificate ARN above. Remove it if you plan to expose your RHDH differently, for example via a CloudFront distribution.
+      alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
+
+
+      # TODO: if needed, set HTTP to HTTPS redirects. Every HTTP listener configured will be redirected to below mentioned port over HTTPS.
+      # alb.ingress.kubernetes.io/ssl-redirect: '443'
+
+
+
+
+  backstage:
+    image:
+      pullSecrets:
+      - rhdh-pull-secret
+    podSecurityContext:
+      # you can assign any random value as fsGroup
+      fsGroup: 2000
+  postgresql:
+    image:
+      pullSecrets:
+      - rhdh-pull-secret
+    primary:
+      podSecurityContext:
+        enabled: true
+        # you can assign any random value as fsGroup
+        fsGroup: 3000
+  volumePermissions:
+    enabled: true
+----
+If you already have the DNS name for your {product-short} instance, specify the value for the `global.host` field in the previous template.
+
+[NOTE]
+====
+Configuring the HTTPS listener with Application Load Balancer (ALB) requires a certificate for your custom domain. In case, the custom domain or certificate is not available, then you can set the `alb.ingress.kubernetes.io/listen-ports` annotation to `[{"HTTP": 80}]` and create a CloudFront distribution using the ALB endpoint as the content origin. Using this approach, you can access the {product-short} using the CloudFront domain name. For more information about creating a CloudFront distribution, see https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-creating.html[Steps for creating a distribution].
+====
+--
+
+. Run the following command in your terminal to deploy {product-short} using the latest version of Helm Chart and using the values.yaml file created in the previous step:
++
+--
+[source]
+----
+helm install rhdh \
+  openshift-helm-charts/redhat-developer-hub \
+  --version 1.0.0-1 \
+  --values /path/to/values.yaml
+----
+In the previous example, `version 1.0.0-1` is used as the latest version of the Helm Chart.
+--
+
+. (Optional) If an ALB or another ingress controller is used to expose your {product-short} instance and you are unaware of the DNS hostname, then identify the provisioned DNS name and update the {product-short} configuration using the following steps:
```

Comment:
This (and the subsequent steps about updating the values file and redeploying the RHDH instance) is no longer needed.

Reply:
Done